omnivoice/cli/demo.py

```
@@ -56,53 +53,53 @@ def get_best_device():
 # Each option is displayed as "English / 中文".
 # The model expects English for accents and Chinese for dialects.
 _CATEGORIES = {
-    "Gender / 性别": ["Male / 男", "Female / 女"],
-    "Age / 年龄": [
-        "Child / 儿童",
-        "Teenager / 少年",
-        "Young Adult / 青年",
-        "Middle-aged / 中年",
-        "Elderly / 老年",
+    _("Gender"): [(_("Male"), "male"), (_("Female"), "female")],
```

Ruff (0.16.2)

[error] Undefined name `_`

(F821)